src/angular/select/select/select.component.scss

```diff
@@ -0,0 +1,97 @@
+@import '../../../angular/styles/common';
+
+.sbb-select {
+  display: flex;
+  cursor: pointer;
+
+  padding-right: pxToRem(12);
+  padding-bottom: pxToRem(10);
+
+  @include sbbStandard() {
```

Comment:
otherwise the styles from sbb-input-element would be applied. I added the following comment:
 // Use variant mixin here to be specific enough to overwrite sbb-input-element styles